submissions/alstep07/task-js-dom/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Task JS DOM</title>
+  <link rel="stylesheet" href="./style.css" type="text/css">
+</head>
+
+<body>
+  <div class="wrapper">
+    <header class="header">
+      <h1 class="header__title">Top-5 Imdb Movies</h1>
+    </header>
+    <nav class="navbar">
+      <h2 class="navbar__header">By Genre</h2>
+      <ul class="navbar__list">
+        <li ><a href="#" class="navbar__item">Action</a></li>
+        <li ><a href="#" class="navbar__item">Drama</a></li>
+        <li ><a href="#" class="navbar__item">Comedy</a></li>
+        <li ><a href="#" class="navbar__item">Western</a></li>
+        <li ><a href="#" class="navbar__item">Romance</a></li>
+        <li ><a href="#" class="navbar__item">Horror</a></li>
+      </ul>
+    </nav>
```

Comment:
Are you try to use [`bem`](https://ru.bem.info/methodology/quick-start/) methodology ?

I think `navbar__item`'s  are `h2` and `ul` tags only.

One more point. Maybe try to render this list with js ?)